### ISSUE TITLE
Шлюзы

### ISSUE THREAD

**Issue by Phelps22** [Tech3434/Techno-Phoenix#25]
ВСЕ шлюзы в начале раунда являются **прозрачными**, пока их не откроют хотя бы один раз. Спрайт при этом у них какой нужно.

**Comment by ak72ti** [comment]
Звучит как бьендопроблемы, баг довольно серьезный, до тебя бы заметили, уверен что это правда не проблема первой загрузки билда?

**Comment by Phelps22** [comment]
Нет, не уверен, но и загрузка билда уже давно не первая. Пережил пару обновлений, вон.
